Exercises-2-Basic/src/exercises/Num5.java

```diff
@@ -0,0 +1,15 @@
+package exercises;
+
+public class Num5 {
+/*
+  5.
+Write a java program that compares two given numbers, and prints the relation between them - >, < , =.
+For example,
+Input: 5 , 10
+Your program should print "5 < 10"
+ */
+	public static void main(String[] args) {
```

Comment:
No solution!

Reply:
Done.

Reply:
It prints the output twice in some cases :-)
It doesn't handle the case where the numbers are euqal.

Reply:
done